### ISSUE TITLE
Use `fastjson` instead of `json`

### ISSUE THREAD

**Issue by hubcio** [iggy-rs/iggy-go-client#9]
It's up to 15 times faster than `json`.
https://github.com/valyala/fastjson

**Comment by eldpcn** [comment]
Good idea, will pick this up when adding HTTP communication (right now it's only TCP and it's still WIP)
